book/quick-brick/native-project-dev.md

```diff
@@ -0,0 +1,132 @@
+# Get ready to work on Quick-Brick
+
+1. [Description](#description)
+2. [Create run a quick-brick bundle from localhost](#localhost_qb)
+3. [iOS/tvOS Quick-Brick](#ios_qb)
+4. [Android and Android T.V](#android_qb)
+
+***
+
+## Description {#description}
+
+This guide will provide information about how to test your react native quick-brick bundle on a native simulator/emulator or actual devices.
```

Comment:
I thought I commented on this but it seems like it slipped through the cracks.

I have a problem with the wording here. I think we should simply say that the QB app is a React Native app, and as such, you can run it with the react-native packager for development / testing purposes.

This is what I meant when I was talking about building natively vs building react-native. It's just one app, the react-native code is part of it. It would be more effective to lean on the react-native terms & wording, instead of adding our own way of saying things and create confusion